domains/seldomgreen.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "seldomgreeen",
+           "email": "72107155+seldomgreeen@users.noreply.github.com",
+           "discord": "761646485334523954"
+        },
+    
+        "record": {
+            "CNAME": "is-a.dev"
```

Comment:
?

Reply:
Yeah this change is fine 👍

Reply:
Its not, you can't point your domain to our domain.

Reply:
Mb, I'm gonna try to find somewhere to host and then I'll let you know what to update it to